### ISSUE TITLE
Cannot boot DPRO image

### ISSUE THREAD

**Issue by fxgallego** [dolphinsmalltalk/Dolphin#757]
**Describe the bug**
Cannot build DPRO image.
I traced the problem to a specific commit c8bce69b1aec83d902dace68a5e941361a4af40b
Remove quoAndRem primitives (contributes to #534)

It's really strange but the build is passing and I can reproduce the problem on 3 different machines.

**To Reproduce**
1. Clone Dolphin repo
2. git lfs pull
3. Download VM
4. Run BootDPRO.cmd

**Expected behavior**
DPRO image build successfully

**Screenshots**
This is on Windows 10 and here is the tail of the console output
```
Loading source package 'Windows Shell' from: C:\Dev\Dolphin\Dolphin\Core\Object Arts\Dolphin\ActiveX\Shell\Windows Shell.pax
Loading source package 'Development System' from: C:\Dev\Dolphin\Dolphin\Core\Object Arts\Dolphin\IDE\Base\Development System.pax
Recompiling references to AbstractLOGFONT (size 28)...
Recompiling references to LOGFONTW (size 92)...
----------------------------------------Unhandled exception - a MessageNotUnderstood('Array does not understand #negated')

Array(Object)>>doesNotUnderstand:
Font>>pointSize:
Font class>>name:pointSize:
SmalltalkWorkspace class>>initialize
SmalltalkWorkspace class(Class)>>initializeAfterLoad
[] in Package>>initializeClasses
Array(ArrayedCollection)>>do:
Package>>initializeClasses
Package>>initializeGlobals
[] in Package>>loadPAX
BlockClosure>>ifCurtailed:
BlockClosure>>ensure:
Package>>loadPAX
Package>>load
PackageManager>>basicInstall:
[] in PackageManager>>install:
BlockClosure>>ifCurtailed:
[] in PackageManager>>install:
OrderedCollection>>do:
[] in PackageManager>>install:
BlockClosure>>ifCurtailed:
BlockClosure>>ensure:
PackageManager>>install:
[] in DolphinProduct>>basicLoad
ExceptionHandler(ExceptionHandlerAbstract)>>markAndTry
[] in ExceptionHandler(ExceptionHandlerAbstract)>>try:
BlockClosure>>ifCurtailed:
BlockClosure>>ensure:
ExceptionHandler(ExceptionHandlerAbstract)>>try:
BlockClosure>>on:do:
[] in DolphinProduct>>basicLoad
PluggableLookupTable(LookupTable)>>at:ifAbsent:
PackageManager>>packageNamed:ifNone:
[] in DolphinProduct>>basicLoad
OrderedCollection>>do:
DolphinProfessional(DolphinProduct)>>basicLoad
DolphinProfessional(DolphinProduct)>>loadShrinkAndCompress
DolphinProfessional(DolphinProduct)>>boot
DolphinProfessional class(DolphinProduct class)>>boot
[] in UndefinedObject>>{unbound}doIt
ExceptionHandler(ExceptionHandlerAbstract)>>markAndTry
[] in ExceptionHandler(ExceptionHandlerAbstract)>>try:
BlockClosure>>ifCurtailed:
BlockClosure>>ensure:
ExceptionHandler(ExceptionHandlerAbstract)>>try:
BlockClosure>>on:do:
UndefinedObject>>{unbound}doIt
Compiler class>>evaluate:for:evaluationPools:logged:ifFail:
Compiler class>>evaluate:for:evaluationPools:logged:
Compiler class>>evaluate:for:logged:
[] in ChunkSourceFiler>>{unbound}fileIn
ExceptionHandler(ExceptionHandlerAbstract)>>{unbound}markAndTry
[] in ExceptionHandler(ExceptionHandlerAbstract)>>{unbound}try:
BlockClosure>>{unbound}ifCurtailed:
BlockClosure>>{unbound}ensure:
ExceptionHandler(ExceptionHandlerAbstract)>>{unbound}try:
BlockClosure>>{unbound}on:do:
ChunkSourceFiler>>{unbound}fileIn
SourceManager>>{unbound}fileInFrom:normalizeLineEndings:
[] in SourceManager>>{unbound}fileIn:normalizeLineEndings:
BlockClosure>>{unbound}ifCurtailed:
BlockClosure>>{unbound}ensure:
SourceManager>>{unbound}fileIn:normalizeLineEndings:
SourceManager>>{unbound}fileIn:
BootSessionManager>>{unbound}main
BootSessionManager(SessionManager)>>{unbound}mainLoopStarted
[] in BootSessionManager(SessionManager)>>{unbound}forkMain
InputState>>{unbound}loopWhile:
InputState>>{unbound}mainLoop
[] in InputState>>{unbound}forkMain
ExceptionHandler(ExceptionHandlerAbstract)>>{unbound}markAndTry
[] in ExceptionHandler(ExceptionHandlerAbstract)>>{unbound}try:
BlockClosure>>{unbound}ifCurtailed:
BlockClosure>>{unbound}ensure:
ExceptionHandler(ExceptionHandlerAbstract)>>{unbound}try:
BlockClosure>>{unbound}on:do:
[] in BlockClosure>>{unbound}newProcess:
----------------------------------------

Shutting down boot session
Boot failed, Code=-1
Presione una tecla para continuar . . .
```

**Comment by blairmcg** [comment]
I made a breaking change, but forgot to push a new release tag, so the VM downloaded by FetchVM won't work. I will do that shortly, but if this happens again you will need to build the VM, as the (working) CI build is doing.

**Comment by blairmcg** [comment]
Unfortunately the release build is not working due to a content type issue with the inno setup installer from its download site; it’s downloading as a text file so cannot be run.
I’m afraid there is no alternative to building the VM yourself at present.

**Comment by blairmcg** [comment]
As the DolphinRelease build is still not working, I've modified the regular appveyor build to upload the VM artefacts, and then manually published them as v7.1.5. You should now find that if you run FetchVM, you will then be able to boot the image. It works for me anyway (whereas it does not on the v7.1.4 VM, because of an incompatible change to the primitive numbers).
In future it will always be possible to download the VM artefacts from the most recent appveyor CI build as well.

**Comment by fxgallego** [comment]
Thanks, It's working now.
